src/applications/vaos/containers/AppointmentListsPage.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+import get from 'platform/utilities/data/get';
+import {
+  fetchConfirmedAppointments,
+  fetchPendingAppointments,
+} from '../actions/appointments';
+
+export class AppointmentListsPage extends Component {
+  componentDidMount() {
+    this.props.fetchConfirmedAppointments();
+    this.props.fetchPendingAppointments();
+  }
+
+  render() {
+    const {
+      confirmed,
+      confirmedLoading,
+      pending,
+      pendingLoading,
+    } = this.props.appointments;
+
+    const loading = confirmedLoading || pendingLoading;
+
+    return (
+      <div className="vads-l-grid-container vads-u-padding-x--2p5 large-screen:vads-u-padding-x--0 vads-u-padding-bottom--2p5">
+        <div className="vads-l-row">
+          <div className="vads-l-col--12 medium-screen:vads-l-col--8 vads-u-margin-bottom--4">
+            <div>
+              <h1>Your Appointments</h1>
+              {loading ? (
```

Comment:
It looks like you have this, and also another nested loading check. Do we need this top level one?

Reply:
Good catch. Removed the nested loading check in favor of this one.